tlsr9/CMakeLists.txt

```diff
@@ -70,50 +89,88 @@ if(CONFIG_BT_B9X OR CONFIG_IEEE802154_TELINK_B9X)
 		common/b9x_rf_power.c
 	)
 endif()
+if(CONFIG_BT_TLX OR CONFIG_IEEE802154_TELINK_TLX)
+	zephyr_library_sources(
+		drivers/${SOC}/aes.c
+		common/b9x_rf_power.c
+	)
+endif()
 
 # ADC driver reference sources
 zephyr_library_sources_ifdef(CONFIG_ADC_TELINK_B9X drivers/${SOC}/adc.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_TELINK_B9X drivers/${SOC}/gpio.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_TELINK_TLX drivers/${SOC}/adc.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_TELINK_TLX drivers/${SOC}/gpio.c)
 
 # USB driver reference sources
 zephyr_library_sources_ifdef(CONFIG_USB_TELINK_B9X drivers/${SOC}/gpio.c)
 zephyr_library_sources_ifdef(CONFIG_USB_TELINK_B9X drivers/${SOC}/usbhw.c)
+zephyr_library_sources_ifdef(CONFIG_USB_TELINK_TLX drivers/${SOC}/gpio.c)
+zephyr_library_sources_ifdef(CONFIG_USB_TELINK_TLX drivers/${SOC}/usbhw.c)
 
 #PM driver dependency
+if (NOT CONFIG_SOC_RISCV_TELINK_TL321X)
 zephyr_library_sources_ifdef(CONFIG_PM drivers/${SOC}/stimer.c)
+endif()
 zephyr_library_sources_ifdef(CONFIG_PM common/sleep.c)
 
 #BLE flash reference sources
-if (CONFIG_IEEE802154_TELINK_B9X)
+if (CONFIG_IEEE802154_TELINK_B9X OR CONFIG_IEEE802154_TELINK_TLX)
 
-zephyr_include_directories(
+zephyr_include_directories_ifdef(CONFIG_IEEE802154_TELINK_B9X
 	ble
-	ble/vendor/controller
+	ble/vendor/controller/b9x
+)
+
+zephyr_include_directories_ifdef(CONFIG_IEEE802154_TELINK_TLX
+	ble
+	ble/vendor/controller/tlx
```

Comment:
Same here, we can use simplified code after some folders/files renaming
```
zephyr_include_directories(
	ble
	ble/vendor/controller/${SOC}
)
```

Reply:
use ${SOC_FAMILY} for `b9x` and `tlx` instead.